packages/playground/blueprints/src/lib/steps/wp-cli.ts

```diff
@@ -31,16 +59,7 @@ export const wpCLI: StepHandler<WPCLIStep, Promise<PHPResponse>> = async (
 	playground,
 	{ command, wpCliPath = '/tmp/wp-cli.phar' }
 ) => {
-	if (!(await playground.fileExists(wpCliPath))) {
-		throw new Error(`wp-cli.phar not found at ${wpCliPath}.
-			You can enable wp-cli support by adding "wp-cli" to the list of extra libraries in your blueprint as follows:
-			{
-				"extraLibraries": [ "wp-cli" ]
-			}
-
-			Read more about it in the documentation.
-			https://wordpress.github.io/wordpress-playground/blueprints/data-format#extra-libraries`);
-	}
+	await installWpCli(playground);
```

Comment:
This will always fetch wp-cli, even if it's already available

Reply:
Also, why fetch it at all here instead of leaving that up to the API consumer? This opens the API up for more scenarios we'll have to maintain and migrate to the PHP library. I understand this is not a convenient constraint to follow, but introducing `installWpCli` opens up API a bit more than I'm comfortable with, e.g. now we have multiple places in the codebase that can perform the installation, the `.phar` file won't be downloaded upfront concurrently with other resources, etc.